### PR TITLE
Telemetry for CLUE on frontend

### PR DESCRIPTION
**Pull request by vandyliu**
# Description:
- Added telemetry for CLUE on frontend (sensors, new file, debugger, button presses)

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/service/telemetryHandlerService.ts [hunk 1]
```diff
@@ -59,6 +69,8 @@ export class TelemetryHandlerService {
             case CONSTANTS.DEVICE_NAME.MICROBIT:
                 this.handleMicrobitButtonPressTelemetry(buttonState);
                 break;
+            case CONSTANTS.DEVICE_NAME.CLUE:
+                this.handleClueButtonPressTelemetry(buttonState)
```
Comment (nasadigital) [inline src/service/telemetryHandlerService.ts]: Perhaps add a break before the default statement, in case we add more cases in the future.

**Review by nasadigital — APPROVED** [review]
That was simpler than I expected. LGTM!

File: src/service/telemetryHandlerService.ts (hunk 1 above)
Comment (vandyliu) [reply src/service/telemetryHandlerService.ts]: good catch!

**Review by andreamah — APPROVED** [review]
🛩✈🚀 Great work!

**Pull request merged**